.travis.yml

```diff
@@ -25,7 +25,7 @@ install:
     - python setup.py install
 
 script:
-    - nosetests conda_smithy
+    - nosetests -v conda_smithy
```

Comment:
I added just to visualize what test was failing and how. We can remove if you want.